src/main/java/org/fundacionjala/coding/denis/FizzBuzz.java

```diff
@@ -0,0 +1,40 @@
+package org.fundacionjala.coding.denis;
+
+import java.util.List;
+import java.util.StringJoiner;
+
+/**
+ * . this is the class principal of FizzBuzz.
+ */
+public class FizzBuzz {
+    private static final int FIZZCONT = 3;
+    private static final int BUZZCONT = 5;
+
+    /**
+     * @param numbers is the data work.
+     * @return the spected result.
+     */
+    public String fizzBuzzCheck(final List<Integer> numbers) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 22 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)